integrations/velero/documentation.yaml

```diff
@@ -0,0 +1,95 @@
+exporter_type: included
+app_name_short: Velero
+app_name: {{app_name_short}}
+app_site_name: Velero
+app_site_url: https://velero.io/
+exporter_name: the Velero Prometheus exporter
+exporter_pkg_name: velero
+exporter_repo_url: https://velero.io/docs/
+minimum_exporter_version: v1.3.0
+additional_prereq_info: |
+  {{app_name_short}} exposes Prometheus-format metrics automatically; you do not
+  have to install it separately. To verify that {{exporter_name}} is emitting
+  metrics on the expected endpoints, do the following:
+
+  1. Set up port forwarding by using the following command:
+
+  <pre class="devsite-click-to-copy">
+  kubectl -n velero port-forward {{cluster_name}} 8085
```

Comment:
I think this was right before? Typically cluster_name refers to the GKE cluster name, does it have a different meaning in the context of velero?

Or at least, when I installed velero to test these instructions locally, I was using `kubectl -n {{namespace_name}} port-forward deploy/velero 8085`

Reply:
That's my mistake, should be good to go.